datastore.go

```diff
@@ -130,7 +148,7 @@ func (d *datastore) QueryNew(q dsq.Query) (dsq.Results, error) {
 	}), nil
 }
 
-func (d *datastore) QueryOrig(q dsq.Query) (dsq.Results, error) {
+func (a *accessor) queryOrig(q dsq.Query) (dsq.Results, error) {
```

Comment:
The exported `QueryOrig` and `QueryNew` are not used in IPFS to my knowledge. I made them private, but feel free to push back if I shouldn't have.